operations/overlay_compute_ngd.yml

```diff
@@ -0,0 +1,38 @@
+id: overlay_compute_ngd
+subclass_of: overlay
+name: overlay compute ngd
+description: This operation computes the normalized Google distance (co-occurrence frequency) in PubMed abstracts and adds virual edges between qnodes AND/OR knodes.
+examples: 
+- input: overlay/messages/07_input_ngd.json
+  output: overlay/messages/08_output_ngd.json
+input_requirements:
+- Input qgraph contains the two qnodes specified in 'subject_qnode_key' and 'object_qnode_key'
+output_guarantees:
+- At least 1 new edge between qnodes
+allowed_changes:
+- add kedges
+- add qedges
+parameters: 
+  type: object
+  properties: 
+    default_value:
+      description: The default value of the normalized Google distance (if its value cannot be determined)
+      type: string
+      example: [0, inf]
+      default: inf
```

Comment:
Shouldn't "example" be an actual single (ideally sibling-consistent) example of what the field might contain? Is this case, default_value would never be a two-item list of [ 0, inf].

OR do I misunderstand what "example" should contain?
More instances of this throughout.

Reply:
My understanding was that example should contain a list of examples. Should it be limited to only one example per parameter? Would it be better to keep it as is and rename to examples instead?

Reply:
Agreed: @finnagin I think a specific value for the example, and minimum and maximum should be provided like [here](https://github.com/NCATSTranslator/OperationsAndWorkflows/blob/fill-bind/operations/filter_results_top_n.yml#L19).

Reply:
Ok I can do that. Thank you!

Reply:
I've changed it now. Is this what you were thinking?